app/components/home/Home.js

```diff
@@ -19,12 +24,69 @@ class Home extends React.Component {
 
 	render() {
 		return (
-			<Text
-				style={ HomeStyles.faqLink }
-				onPress={ () => this.props.navigation.navigate("FaqScreen") }
-			>
-				FAQs
-			</Text>
+			<View style={ HomeStyles.container }>
+				<View style={ HomeStyles.topBar }>
+					<Icon
+						name="notifications"
+						color="#696969"
+						size={ 35 }
+						onPress={ () => this.props.navigation.navigate("AlertsScreen") }
+					/>
+					<Image resizeMode="contain" style={ HomeStyles.logo } source={ logo } />
+					<Icon
+						name="settings"
+						color="#696969"
+						size={ 35 }
+						onPress={ () => this.props.navigation.navigate("SettingsScreen") }
+					/>
+				</View>
+				<Text
+					style={ HomeStyles.welcomeMessage }
+				>
+					Welcome to the Water Festival app!
+				</Text>
+				<Button
+					onPress={ () => this.props.navigation.navigate("AllActivitiesScreen") }
+					activeOpacity={ 1 }
+					containerStyle={ HomeStyles.button }
+					style={ HomeStyles.buttonText }
+				>
+					Choose Your Activities
+				</Button>
+				<Button
+					onPress={ () => this.props.navigation.navigate("MyActivitiesScreen") }
+					activeOpacity={ 1 }
+					containerStyle={ HomeStyles.button }
+					style={ HomeStyles.buttonText }
+				>
+					My Schedule
+				</Button>
+				<Button
+					onPress={ () => this.props.navigation.navigate("GroundwaterDetectivecreen") }
```

Comment:
Missing the S in Screen but I guess it doesn't matter cuz it hasnt been implemented yet